WooCommerce/Classes/POS/Models/PointOfSaleAggregateModel.swift

```diff
@@ -429,21 +428,6 @@ extension PointOfSaleAggregateModel {
         })
         await startPaymentWhenCardReaderConnected()
     }
-
-    func publishOrderState() {
-        orderController.orderStatePublisher
-            .map { $0.externalState }
-            .sink(receiveValue: { [weak self] orderState in
-                self?.orderState = orderState
-            })
-            .store(in: &cancellables)
-
-        orderController.orderStatePublisher
-            .sink(receiveValue: { [weak self] internalOrderState in
-                self?.internalOrderState = internalOrderState
-            })
-            .store(in: &cancellables)
-    }
```

Comment:
Very nice to just "observe" the state we're interested in, without having to write all this code 💯